master/buildbot/buildslave/ec2.py

```diff
@@ -287,6 +279,13 @@ def start_instance(self, build):
         else:
             return threads.deferToThread(self._start_instance)
 
+    def _failed_to_start(self):
+        log.msg('%s %s failed to start instance %s (%s)' %
+                (self.__class__.__name__, self.slavename,
+                    self.instance.id, self.instance.state))
+        raise LatentBuildSlaveFailedToSubstantiate(
+            self.instance.id, self.instance.state)
+
```

Comment:
The `instance` member is not part of the abstract class...

I'd appreciate a suggestion on how to deal with it :)